test/setup.sh

```diff
@@ -196,7 +208,14 @@ if [ "$memserver" ]; then
 	if [ "$fam_path" ]; then
          	fam_path_ARG="-f ${fam_path}"		
 	fi
-	./memory_server -a ${memserver} -r ${memrpcport} -l ${libfabricport} -p ${provider} $fam_path_ARG &
+	if [ "$memserver_id" ]; then
+         	memserver_id_ARG="-m ${memserver_id}"		
+	fi
+	if [ "$libfabricport" ]; then
```

Comment:
@faizan-barmawer should we remove other parameters as part of this PR ?

Reply:
It will be good if we do this change now, instead of doing it later. We can fetch all the parameters for memory server start service from the config file, instead of command line arguments. 
We can keep only memory server id parameter to the memory_server command, Let the service fetch all other values from the configuration file.